pkg/features/httproute.go

```diff
@@ -157,6 +160,11 @@ var (
 		Name:    SupportHTTPRouteRequestMultipleMirrors,
 		Channel: FeatureChannelStandard,
 	}
+	// HTTPRouteRequestPercentageMirrorFeature contains metadata for the HTTPRouteRequestMultipleMirrors feature.
+	HTTPRouteRequestPercentageMirrorFeature = Feature{
+		Name:    SupportHTTPRouteRequestPercentageMirror,
+		Channel: FeatureChannelStandard,
```

Comment:
Even though this GEP aims at graduation, I'd put experimental here, and eventually flip it with the feature graduation.